### DISCUSSION TITLE
how to run a unikernel

### DISCUSSION THREAD

**Discussion by qsmen** [unikraft/unikraft#948] [Q&A] [answered]
I create an unikernel of kvm-type . How can I run it in a kvm-type VM? qemu -kernel unikernel?

**Comment by StefanJum** [comment, marked as answer]
@qsmen we usually use [Discord](https://bit.ly/UnikraftDiscord) for support, it's easier to get a response there.
For running Unikraft applications you can find a section on the [Unikraft website](https://unikraft.org/docs/usage/make_build/#run-the-application).

**Reply by qsmen** [reply]
thank you. the unikerenel can be used as the -kernel parameter in qemu ...
